src/app/shell/personal-cabinet/provider/create-workshop/create-about-form/create-about-form.component.ts

```diff
@@ -98,21 +97,32 @@ export class CreateAboutFormComponent implements OnInit {
   * This method fills in the info from provider to the workshop if check box is checked
   */
   useProviderInfo(): void {
-    this.useProviderInfoCtrl.valueChanges.subscribe((val) => {
-      if (val) {
+    this.useProviderInfoCtrl.valueChanges.subscribe((useProviderInfo: boolean) => {
+      if (useProviderInfo) {
         this.AboutFormGroup.get('email').setValue(this.provider.email);
         this.AboutFormGroup.get('title').setValue(this.provider.fullTitle);
         this.AboutFormGroup.get('phone').setValue(this.provider.phoneNumber);
         this.AboutFormGroup.get('website').setValue(this.provider.website);
         this.AboutFormGroup.get('facebook').setValue(this.provider.facebook);
         this.AboutFormGroup.get('instagram').setValue(this.provider.instagram);
       } else {
-        this.AboutFormGroup.get('email').setValue('');
-        this.AboutFormGroup.get('phone').setValue('');
-        this.AboutFormGroup.get('website').setValue('');
-        this.AboutFormGroup.get('facebook').setValue('');
-        this.AboutFormGroup.get('instagram').setValue('');
+        this.AboutFormGroup.get('email').reset();
```

Comment:
I am wondering if we can emit reset directly from form to avoid resetting each field